sd-2019-stack-overflow-frontend/src/view/AskQuestionModal.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+const AskQuestionModal = ( {askQuestionModalClass, onCloseAskQuestionModal, newQuestion, newTag,
+    onChangeNewQuestionProperty, onChangeNewTag, onCreateNewTag, onDeleteNewTag, onCreateQuestion} ) => (
+    <div className={askQuestionModalClass}>
```

Comment:
It is better to get the `active` flag as an input instead of the class. Then you can do the following: `className={"modal" + (active ? " is-open" : "")}`. This way the smart component does not know about your CSS.